app/containers/Forms/EmissionSourceFields.tsx

```diff
@@ -31,12 +31,17 @@ const EmissionSourceFields: React.FunctionComponent<FieldProps> = ({
         }
       `}</style>
       <style jsx>{`
-        h6 {
+        .unit {
           text-align: center;
```

Comment:
Corresponds to changing the above tags from `h6` --> `p` while accounting for their specific styling.